core/BUILD.bazel

```diff
@@ -63,3 +63,17 @@ java_library(
     ],
 )
 
+# Mirrors the dependencies included in the artifact on Maven Central for usage
+# with maven_install's override_targets. Purposefully does not export any
+# symbols, as it should only be used as a dep for pre-compiled binaries on
+# Maven Central.
+java_library(
+    name = "core_maven",
+    visibility = ["//visibility:public"],
+    runtime_deps = [
```

Comment:
Why is this runtime_deps?

Reply:
Did the documentation not explain it?
> Purposefully does not export any symbols, as it should only be used as a dep for pre-compiled binaries on Maven Central.

I didn't want Bazel users to get lazy and use this "mega" target instead of the more fine-grained targets.